### DISCUSSION TITLE
Short Description/Description mandatory?

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#3781] [Ideas]
By default, these two fields in the product editing form are mandatory and have been so since the beginning. I encountered situations when only one of them was necessary, but I had to complete the other one with a &nbsp because saving the product does not allow it to be empty. This is how I ended up disabling the mandatory filling up and changing the templates if these fields are empty so that they no longer display the blocks that contain them in Frontend.

Here is a concrete situation when these fields shouldn't be mandatory

1 x Product Configurable for size only. It contains the product images, descriptions, category.
4 x Simple Products for S, M, L, XL sizes. They are only used for inventory control, no description, no images.

If the Associated Products are created in the Configurable Product then they will take over the Short Description/Description from the Configurable Product. This is not good, because it unnecessarily loads the database with data in this case. So the Simple Products must be created before the Configurable Product, and Short Description/Description are not requested, and OpenMage does not offer the feature to change them when a variant is selected. The fact that these fields are mandatory is annoying because when you want to save a simple product you have to fill in something there.

What do you think? Should the Short Description/Description fields no longer be mandatory?

![screenshot](https://github.com/OpenMage/magento-lts/assets/8360474/f99a9719-94ae-433f-9613-1f40f9b6947d)

**Comment by DegrizNet** [comment]
No they are not mandatory.. I would change that and also order short description and then description is much better. Alos when talking about fields.. :D .. it would be great that on creating configurable product option use defalut on label would be selected by default.

https://magento.stackexchange.com/questions/102330/set-the-checkbox-value-to-checked-by-default-for-super-product-attribute-configu

**Reply by addison74** [reply]
The order of the attributes is coming from the Default set, but that is another discussion. What you see in the image is from production, where groups of attributes have been created in a set to ease the work and make the filling in intuitive The two fields should not be mandatory and small changes must be done in a few templates.

**Comment by sreichel** [comment]
You can make them not required in "Catalog -> Manage Attributes" page.

**Reply by addison74** [reply]
That's what I did. It would have been good if in the installation process they were not set as required.
